sdk/metricsadvisor/Azure.AI.MetricsAdvisor/CHANGELOG.md

```diff
@@ -25,6 +26,7 @@
 - Renamed class `AzureDataLakeStorageGen2DataFeedSource` to `AzureDataLakeStorageDataFeedSource`. Similarly, `DataSourceType.AzureDataLakeStorageGen2` has been renamed to `AzureDataLakeStorage`.
 - Renamed class `FeedbackType` to `MetricFeedbackKind`. Similarly, `GetAllFeedbackOptions.FeedbackType` has been renamed to `FeedbackKind`, and `MetricFeedback.Type` has been renamed to `Kind`.
 - Renamed class `PeriodType` to `MetricPeriodType`.
+- Renamed class `FeedbackDimensionFilter` to `GetAllFeedbackFilter` and moved it to the namespace `Azure.AI.MetricsAdvisor`.
```

Comment:
the class name `GetAllFeedbackFilter` is weird because of the verb. Is this a convention because this class is related to the `Options` one?  can you just leave the previous name? or change to something like `AllFeedbackFilter`, `AllFeedbackFilterOptions` `FeedbackFilter`, etc?

Reply:
Updated.